inject.js

```diff
@@ -123,11 +127,14 @@ module.exports = function (blocks, frame, codec, file, cache) {
         })
       )).catch((err) => cb(err))
       .then(() => cb(null))
+    },
+    close: function (cb) {
+      cb()
```

Comment:
is this meant to do anything or just provide a consistent api? if it's to make this module consistent with some interface, maybe add a comment saying "nothing to do when closing". anyway, seems innocent, but then there's other changes in this pull request that seem unrelated.